README.md

```diff
@@ -281,7 +287,211 @@ Examples of WordPress block markup and the associated data structure returned by
 </tr>
 </table>
 
-## Preact example
+### GraphQL
+
+The GraphQL API, requires some setup before it can be it can be used.
+
+#### Setup
+
+The Block Data API integrates with **WP-GraphQL** to provide a GraphQL API as well. As a result, it is necessary to have WP-GraphQL installed, to activate the GraphQL API. To do so, follow the instructions mentioned [here](https://www.wpgraphql.com/docs/quick-start#install). 
+
+Once WP-GraphQL has been installed and setup, a new field called `blocksData` will show up under categories that support `ContentNodes` like posts, pages, etc. 
+
+#### Usage
+
+The `blocksData` field would be the field through which block data would be returned under a category that supports it. 
+
+An example of what a query would look like for a post:
```

Comment:
Category has a different meaning in a WP context.
```suggestion
The `blocksData` field provides block data for post types that support it. Here is an example query:
```

Reply:
Fixed in https://github.com/Automattic/vip-block-data-api/commit/fac6ebf59e071e5db10234cabe951f7db1d4894f.